SIPS/sip-124.md

```diff
@@ -0,0 +1,55 @@
+---
+sip: 124
+title: Multi-file Snap checksum
+status: Final
+author: Olaf Tomalka (@ritave)
+created: 2023-11-28
+---
+
+## Abstract
+
+This SIP describes an algorithm used to checksum a snap in a reproducible manner in a way that includes all files required to run the snap.
+
+## Specification
+
+> Indented sections like this are considered non-normative.
+
+### Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and
+"OPTIONAL" written in uppercase in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt)
+
+### Steps
+
+#### Checksum `snap.manifest.json`
+
+> This algorithm works on `snap.manifest.json` version `0.1`.
+
+<!-- TODO(ritave): Extract the algorithm from fast-json-stable-stringify and put it here -->
```

Comment:
Feels weird to have a comment in the spec, can we either put the algorithm in here now or open an issue to add in later?